docs/reference/modules/discovery/bootstrapping.asciidoc

```diff
@@ -10,11 +10,20 @@ data folder and freshly-started nodes that are joining an existing cluster
 obtain this information from the cluster's elected master. 
 
 The initial set of master-eligible nodes is defined in the
-<<initial_master_nodes,`cluster.initial_master_nodes` setting>>. This is a list
-of the <<node.name,node names>> or IP addresses of the master-eligible nodes in
-the new cluster. If you do not configure `node.name` then it is set to the
-node's hostname, so in this case you can use hostnames in
-`cluster.initial_master_nodes` too.
+<<initial_master_nodes,`cluster.initial_master_nodes` setting>>. This should be
+set to a list containing one of the following items for each master-eligible
+node:
+
+- The <<node.name,node name>> of the node.
+- The node's hostname if `node.name` is not set, because `node.name` defaults
```

Comment:
Having this as a separate item will I think only lead to confusion (which you already go at  great length of clearing up in `discovery-settings.asciidoc`) that this thing will somehow be resolved similar to `seed_hosts`.